src/repositories/jobRepository.ts

```diff
@@ -35,7 +41,7 @@ export class JobRepository extends BaseRepository {
       update,
       `Mongo Timeout Error: Timed out while updating success status for jobId: ${id}`
     );
-    if (bRet) {
+    if (bRet && shouldNotifySqs) {
```

Comment:
NIT: I believe line 45 (the `this.notify` call) now has a type error since the type of `id` is now `string | mongodb.ObjectID`. 

One approach you could take is to use the `objectId` variable that you defined on line 30, and call its `toString` method to get the string ID. It seems a bit wonky, but this would guarantee that the value will be a string and not potentially the `mongodb.ObjectID` type. 

ex.

```ts
      await this.notify(objectId.toString(), c.get('jobUpdatesQueueUrl'), JobStatus.completed, 0);
```